### PR TITLE
RE2022-117: Simplify method sigs for data product processes

### PR DESCRIPTION
**Pull request by MrCreosote**

### CODE REVIEW

File: src/service/models.py [hunk 1]
```diff
@@ -432,7 +432,27 @@ class ProcessType(str, Enum):
     """ A seletion process. """
 
 
-class DataProductProcess(ProcessAttributes):
+class DataProductProcessIdentifier(BaseModel):
```
Comment (dauglyon) [inline src/service/models.py:435]: Idk if I love this verbiage, but I get it, once I understood it.

Comment (dauglyon) [reply src/service/models.py:435]: just a little odd to have the type/data product ID also in there.

DataProductProccessLocator? Idk.

**Review by dauglyon — APPROVED** [review]
approved regardless

File: src/service/models.py (hunk 1 above)
Comment (MrCreosote) [reply src/service/models.py:435]: yeah, not a huge fan either but it describes what it does and i couldn't think of anything better. If you have better ideas I'm happy to consider it

**Pull request merged**